.travis.yml

```diff
@@ -1,12 +1,11 @@
 language: objective-c
+osx_image: xcode9.2
```

Comment:
We could select xcode9.3 here as well, right?

Reply:
Yes, but then Swift 4.1 gets selected, which results in deprecation warnings.

Reply:
Alright. Good point. Let's to that when we migrate to 4.1 then. Thanks.